### DISCUSSION TITLE
is next/image compatible with serverless build target?

### DISCUSSION THREAD

**Discussion by keyboSlice** [vercel/next.js#25934] [Help] [answered]
Is it possible to use next/image with the Serverless build target without writing a lambda function to process the images myself? 

Looking through the build output it doesn't appear that any sort of image processing handler is generated, and there is no images manifest. 

This makes me think that next/image isn't compatible with this build target, is anyone is able to confirm please?

**Comment by ascorbic** [comment, marked as answer]
Yes, you either need to write a function to process it yourself, or you need to switch to a different provider such as Cloudinary or Imgix. If you go the route of creating your own function, a starting point could be the function used in the Netlify Next.js plugin: https://github.com/netlify/netlify-plugin-nextjs/blob/main/src/lib/templates/imageFunction.js

**Reply by keyboSlice** [reply]
I thought as much, just wanted to be sure it wasn't buried somewhere other than the serverless output directory, thanks for clarifying though
